terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java

```diff
@@ -0,0 +1,207 @@
+            assertThat(form.getField("paymentAccount"), is(
+                    "Current Account：12345678"));
+            assertThat(form.getField("childCount"), is("3"));
+            assertThat(form.getField("tourName"), is(
+                    "【おすすめ】Terasolunaツアー(6泊7日)"));
+            assertThat(form.getField("accomName"), is("TERASOLUNAホテル第一荘"));
+            assertThat(form.getField("customerKana"), is("キムラ　タロウ"));
+            assertThat(form.getField("customerTel"), is("111-1111-1111"));
+            assertThat(form.getField("adultUnitPrice"), is("75000"));
+            assertThat(form.getField("reservedDay"), is("2019/02/21"));
+            assertThat(form.getField("conductor"), is("Yes"));
+            assertThat(form.getField("tourAbs"), is(
+                    "そこは別天地、静寂と湯けむりの待つ宿へ… 詳しい情報はお取り合わせをお願い致します。"));
+            assertThat(form.getField("customerAdd"), is("千葉県八千代市上高野"));
+            assertThat(form.getField("customerJob"), is("プログラマ"));
+            assertThat(form.getField("tourDays"), is("7"));
+            assertThat(form.getField("depDay"), is("2019/03/31"));
+            assertThat(form.getField("customerName"), is("木村　太郎"));
+            assertThat(form.getField("childUnitPrice"), is("37500"));
+            assertThat(form.getField("depName"), is("北海道"));
+            assertThat(form.getField("customerBirth"), is("1975/01/05"));
+            assertThat(form.getField("arrName"), is("北海道"));
+            assertThat(form.getField("customerMail"), is("tarou@example.com"));
+            assertThat(form.getField("adultCount"), is("5"));
+            assertThat(form.getField("customerCode"), is("00000001"));
+            assertThat(form.getField("reserveNo"), is("00000001"));
+            assertThat(form.getField("remarks"), is("特になし"));
+            assertThat(form.getField("accomTel"), is("018-123-4567"));
+            assertThat(form.getField("customerPost"), is("276-0022"));
+            assertThat(form.getField("printDay"), is("2019/03/06"));
+            assertThat(form.getField("adultPrice"), is("375000"));
+            assertThat(form.getField("childPrice"), is("112500"));
+            assertThat(form.getField("sumPrice"), is("487500"));
+            assertThat(form.getField("paymentTimeLimit"), is("2019/03/24"));
+
+            assertThat(response.getHeader("Content-Disposition"), is(
+                    "attachment; filename=reservationReport.pdf"));
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+    }
+
+    @Test
+    public void testPrepareResponse() {
+        try {
+            reservationReportPdfStamperView.prepareResponse(request, response);
+            assertThat(response.getCharacterEncoding(), is(
+                    StandardCharsets.UTF_8.name()));
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+    }
+}
```

Comment:
- [ ] You should import class `com.lowagie.text.pdf.PdfStamper`.

https://github.com/terasolunaorg/terasoluna-tourreservation/blob/164a01ef7936041b8abe7f287f308a0c633f422f/terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java#L52

Reply:
131e3dec86748ee4026b19192fc9a06b14bd0ee8

I fixed in above commit .